core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/CodeSystemRestService.java

```diff
@@ -44,7 +44,7 @@
 /**
  * @since 1.0
  */
-@Tag(description = "Resources", name = "resources")
+@Tag(description = "CodeSystems", name = "codesystems")
```

Comment:
This can be changed to a constant reference as well, unless `CoreApiConfig` is not reachable from here:

```suggestion
@Tag(description = "CodeSystems", name = CoreApiConfig.CODESYSTEMS)
```